servicemesh-install/topics/install.adoc

```diff
@@ -10,12 +10,18 @@ The {ProductShortName} installation process introduces a Kubernetes operator to
 == Installing the operator
 The following steps will install the {ProductShortName} operator into an existing {product-title} 3.10 installation; they can be executed from any host with access to the cluster.  Please ensure you are logged in as a cluster admin before executing the following commands.
 
-You can find the https://github.com/Maistra/openshift-ansible/tree/maistra-0.1.0-ocp-3.1.0-istio-1.0.0/istio[operator templates on GitHub].
+You can find the https://github.com/Maistra/openshift-ansible/tree/maistra-0.2.0-ocp-3.1.0-istio-1.0.2/istio[operator templates on GitHub].
 
 ```
 $ oc new-project istio-operator
 $ oc new-app -f istio_product_operator_template.yaml --param=OPENSHIFT_ISTIO_MASTER_PUBLIC_URL=<master public url>
 ```
+[NOTE]
+.OpenShift Master Public URL
+=====================
```

Comment:
Four "=" signs

Remove the note label.

Rewrite to not use "should."

s/your OpenShift Console, this/{product-title}. This

Reply:
This should now be done